jobs/bbr-postgres-db/templates/restore.sh.erb

```diff
@@ -23,6 +23,10 @@ export_as_private_temp_file "$JOB_DIR/config/pgpass" PGPASSFILE
 export_as_private_temp_file "$JOB_DIR/config/client_certificate_key" PGSSLKEY
 <% end %>
 
```

Comment:
Please just add a variable initialization:
```
singeltrans=
```
otherwise the `-u` will cause a
```
./bbr/restore: line 33: singeltrans: unbound variable
``` 
when the property is false.

Reply:
ugh i feel like such a noob now..
i just wanted a quick fix without any testing. my bad
i added proper var names and init val.
i first tought it would be maby nice to add an if else for it. but this is a bit cleaner i guess